### PR TITLE
Milestone 5: Add design

### PR DESCRIPTION
**Pull request by descholar-ceo**
## What does this PR do?
It adds the the ncessary designs to the project

## Tasks to be completed on this PR
- [x] Install `node-sass` dependency to support sass
- [x] Add other features which act as placeholders
- [x] Style the whole app, to make it look like the given design
- [x] Polish codes and fix stylelint errors

## Deployment
[View BookStore on heroku](https://descholar-bookstore-react.herokuapp.com/)

### CODE REVIEW

File: src/containers/BooksForm.js [hunk 1]
```diff
@@ -23,13 +23,13 @@ const BooksForm = ({ createBook }) => {
 
   return (
     <div>
-      <h2>Create a book</h2>
-      <form onSubmit={event => handleSubmit(event)}>
+      <h2 className="new-book-head">Add a new book</h2>
+      <form className="display-grid" onSubmit={event => handleSubmit(event)}>
```
Comment (mosaaleb) [inline src/containers/BooksForm.js:27]: - [x] **[OPTIONAL]** The form used to create a book should have validations, you should not be able to create an empty book. We can add a `validate` function before creating the book and maybe display errors in the form to inform the user.

**Review by mosaaleb — CHANGES_REQUESTED** [review]
#### Changes required

Excellent job is done @descholar-ceo :clap:. You have handled all the design requirements. The style looks clean and meets the requirements. Yet, since this is the last milestone per our application, we need to start working on the README.md file and add the necessary information about your application.



Kindly take a look at my comment below.

Please handle this requirement and submit a new code review.

#### Muhammad Ebeid

File: src/containers/BooksForm.js (hunk 1 above)
Comment (descholar-ceo) [reply src/containers/BooksForm.js:27]: I didn't implement that but I used the `required` flag on input field,

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @descholar-ceo :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**